scripts/backup

```diff
@@ -18,45 +20,37 @@ ynh_abort_if_errors
 #=================================================
 # LOAD SETTINGS
 #=================================================
-ynh_script_progression --message="Loading installation settings..." --time --weight=1
+ynh_script_progression --message="Loading installation settings..." --weight=1
```

Comment:
I've just launched package_linter, why did you delete all these '--time' option, package_linter is not happy :(

Reply:
As indicate in example_ynh:

https://github.com/YunoHost/example_ynh/blob/429b410dc591626b99b7972796f2dbfd1be0b040/scripts/install#L49-L56

--time is a packager option, This option should be removed before releasing your app.

You only keep it during one package_check to be able to provide a `weigth` value for each ynh_script_progression

So if you want to adjust the weight of each `ynh_script_progression`, you can add them back and also `--last` for the last  `ynh_script_progression`, launch a package check, retrieve the value of each step, adjust the `weight` for each  `ynh_script_progression` and remove `--time` and `--last`

Reply:
Ok thanks !